src/schema/firefox-schemas-import.js

```diff
@@ -8,6 +8,12 @@ import request from 'request';
 import tar from 'tar';
 
 const FLAG_PATTERN_REGEX = /^\(\?[im]*\)(.*)/;
+export const FLAG_PATTERN_REWRITES = {
+  '(?i)^\\{[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}\\}$':
+    // eslint-disable-next-line max-len
+    '^\\{[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}\\}$',
+  '(?i)^[a-z0-9-._]*@[a-z0-9-._]+$': '^[a-zA-Z0-9-._]*@[a-zA-Z0-9-._]+$',
+};
```

Comment:
Can you add a example of what the regex matches as a comment or maybe put the separate regexes into constants to make the object easier to read? Not sure what I prefer tbh… I think I am generally missing some details of why the rewrite is even needed. Maybe a comment solves my problem too :)